### DISCUSSION TITLE
HELP NOOB PLZ! Not even sure what to ask?

### DISCUSSION THREAD

**Discussion by beefqueef** [Cookie-AutoDelete/Cookie-AutoDelete#1087] [Q&A / Support]
Hello, I dont understand any of this stuff. its way over my head. heck i dont even understand github or its use. But with this extension, i need help doing something which is probably super simple, i just dont even know where to start.

I just want to keep a few sites, Youtube, FB, etc, from having their cookies be cleaned. The rest of my cookies can be deleted. I just wanna keep a select few from being deleted. How do i set that up? please help!

**Comment by kennethtran93** [comment]
To better assist you, could you tell me which browser you are using?

**Comment by kennethtran93** [comment]
Just checking in - do you still require assistance?
